### PR TITLE
[zh] Sync #13284 into Chinese with missed translation

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #13284 into Chinese with missed translation

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/blog/2021/better-external-authz/index.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ Istio 的授权策略为网格中的服务提供访问控制。它速度快、
 [嵌套 JWT 声明支持](/zh/docs/tasks/security/authorization/authz-jwt/)等等。
 这些特性提高了授权策略的灵活性，但仍有许多场景无法通过该模型支持，例如：
 
-- 您拥有自己的内部授权系统，该系统无法轻松迁移或替换至授权策略中。
+- 您拥有自己的内部授权系统，该系统无法轻松的被迁移或替换至授权策略中。
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:20]:
```suggestion
- 您拥有自己的内部授权系统，该系统无法轻松地被迁移或替换到授权策略中。
```

File: content/zh/blog/2021/better-external-authz/index.md [hunk 2]
```diff
@@ -63,20 +63,19 @@ metadata:
...
   namespace: istio-system
 spec:
-  # The selector applies to the ingress gateway in the istio-system namespace.
+  # selector 适用于 istio-system 命名空间中的入口网关。
   selector:
     matchLabels:
       app: istio-ingressgateway
-  # The action "CUSTOM" delegates the access control to an external authorizer, this is different from
-  # the ALLOW/DENY action that enforces the access control right inside the proxy.
+  # “CUSTOM” 操作将 action 委托给外部授权者，
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:70]:
```suggestion
  # “CUSTOM” 操作将访问控制委托给外部授权者，
```

File: content/zh/blog/2021/better-external-authz/index.md [hunk 3]
```diff
@@ -63,20 +63,19 @@ metadata:
...
+  # 这与在代理内部强制执行访问控制权的 ALLOW/DENY 操作不同。
   action: CUSTOM
-  # The provider specifies the name of the external authorizer defined in the meshconfig, which tells where and how to
-  # talk to the external auth service. We will cover this more later.
+  # provider 指定在 meshconfig 中定义的外部授权者的名称，
+  # 它告诉在哪里以及如何与外部身份验证服务通信。我们稍后会详细介绍这一点。
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:74]:
```suggestion
  # 从这个名称可以告知在哪里以及如何与外部身份验证服务通信。我们稍后会详细介绍这一点。
```
which 指的是哪个呢

File: content/zh/blog/2021/better-external-authz/index.md [hunk 4]
```diff
@@ -63,20 +63,19 @@ metadata:
...
   provider:
     name: "my-ext-authz-service"
-  # The rule specifies that the access control is triggered only if the request path has the prefix "/admin/".
-  # This allows you to easily enable or disable the external authorization based on the requests, avoiding the external
-  # check request if it is not needed.
+  # 该规则指定只有请求路径有前缀 “/admin/” 时才触发访问控制。
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:77]:
```suggestion
  # 这条规则指定只有请求路径有前缀 “/admin/” 时才触发访问控制。
```

File: content/zh/blog/2021/better-external-authz/index.md [hunk 5]
```diff
@@ -87,15 +86,16 @@ spec:
...
-  # ext-authz filter gRPC API. The other supported type is the Envoy ext-authz filter HTTP API.
+  # “envoyExtAuthzGrpc” 字段指定 Envoy ext-authz 过滤器 gRPC API 实现的外部授权服务的类型。
+  # 另一种支持的类型是 Envoy ext-authz 过滤器 HTTP API。
   # See more in https://www.envoyproxy.io/docs/envoy/v1.16.2/intro/arch_overview/security/ext_authz_filter.
+  # 更多信息请参见 https://www.envoyproxy.io/docs/envoy/v1.16.2/intro/arch_overview/security/ext_authz_filter。
   envoyExtAuthzGrpc:
-    # The service and port specifies the address of the external auth service, "ext-authz.istio-system.svc.cluster.local"
-    # means the service is deployed in the mesh. It can also be defined out of the mesh or even inside the pod as a separate
-    # container.
+    # service 和 port 指定外部auth服务的地址，
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:96]:
```suggestion
    # service 和 port 指定外部 auth 服务的地址，
```

File: content/zh/blog/2021/better-external-authz/index.md [hunk 6]
```diff
@@ -87,15 +86,16 @@ spec:
...
+    # “ext-authz.istio-system.svc.cluster.local” 表示该服务部署在mesh中。
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:97]:
```suggestion
    # “ext-authz.istio-system.svc.cluster.local” 表示该服务部署在网格中。
```

File: content/zh/blog/2021/better-external-authz/index.md [hunk 7]
```diff
@@ -194,7 +194,7 @@ spec:
   selector:
     app: httpbin-with-opa
 ---
-# Define the service entry for the local OPA service on port 9191.
+# 定义 9191 端口本地 OPA 服务的服务条目。
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:197]:
```suggestion
# 在 9191 端口为本地 OPA 服务定义服务条目。
```

File: content/zh/blog/2021/better-external-authz/index.md [hunk 8]
```diff
@@ -116,7 +116,7 @@ extensionProviders:
 
 有关详细信息，请参阅 [Better External Authorization 设计文档](https://docs.google.com/document/d/1V4mCQCw7mlGp0zSQQXYoBdbKMDnkPOjeyUb85U07iSI/edit#)。
 
-## OPA 示例 {#example-with-opa}
+## OPA 示例  {#example-with-opa}
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:119]: line 119 的修改不是很必要，有一个空格应该够了吧

File: content/zh/blog/2021/better-external-authz/index.md [hunk 9]
```diff
@@ -3,11 +3,11 @@ title: 更好的外部授权方式集成
 subtitle: 使用 AuthorizationPolicy 将外部授权系统（例如 OPA、oauth2-proxy 等）与 Istio 进行集成
 description: AuthorizationPolicy 现在支持以 CUSTOM 自定义方式委托外部系统进行授权操作。
 publishdate: 2021-02-09
-attribution: Yangmin Zhu (Google)
+attribution: Yangmin Zhu (Google); Translated by Wilson Wu (DaoCloud)
```
Comment (windsonsea) [inline content/zh/blog/2021/better-external-authz/index.md:6]: This change to attribution shall be completed in the first localization. An update to this attribution may cause question to check.

I have verified the [history](https://github.com/istio/istio.io/commits/master/content/zh/blog/2021/better-external-authz/index.md) and confirmed that both the author and maintainer listed are Wilson, no any other person involved. Everything appears to be correct.

Comment (wilsonwu) [reply content/zh/blog/2021/better-external-authz/index.md:6]: Thanks, it is myself too.

File: content/zh/blog/2021/better-external-authz/index.md (hunk 8 above)
Comment (wilsonwu) [reply content/zh/blog/2021/better-external-authz/index.md:119]: Just want to keep same format.

**Comment by windsonsea** [comment]
This change is relatively longer, so we give some more time to fly 😄

**Review by windsonsea — APPROVED** [review]
lgtm

**Comment by windsonsea** [comment]
Don't know why the lint-check failed.

/retest

**Pull request merged**